src/components/NotificationError/NotificationError.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import {
+  Dispatch,
+  FC,
+  SetStateAction,
+  useEffect,
```

Comment:
Consider using useCallback for the setError function to avoid unnecessary re-renders 🧐